bindata/v3.11.0/openshift-apiserver/deploy.yaml

```diff
@@ -95,6 +98,10 @@ spec:
             path: healthz
       terminationGracePeriodSeconds: 70 # a bit more than the 60 seconds timeout of non-long-running requests
       volumes:
+      - name: node-pullsecrets
+        hostPath:
+          path: /var/lib/kubelet/config.json
+          type: File
```

Comment:
this is not optional. https://github.com/openshift/enhancements/pull/136/files#diff-a3521cf1ef50ae6c95913acbdca6106dR176 only states existence for worker nodes, but not master.

Reply:
Thanks for your comment @sttts. It made us chase the needed info down the road and yes, the pull secret file is always present on master nodes as well. The enhancement proposal got updated to include this missing information.

Reply:
:+1: